### PR TITLE
Highlight and expand corresponding reports when clicking on a log table row

### PR DESCRIPTION
**Pull request by niconoir**

### CODE REVIEW

File: src/components/ReportViewer/log-table.js [hunk 1]
```diff
@@ -84,15 +86,36 @@ const LogTable = ({ logs }) => {
...
+    const handleRowClick = (data) => {
+        setSelectedRowIndex(data.index);
+        onRowClick(data);
+    };
+
+    const rowStyleFormat = (row) => {
+        if (row.index < 0) return;
+        if (selectedRowIndex === row.index) {
+            return {
+                backgroundColor: 'lightblue',
```
Comment (laurgarn) [inline src/components/ReportViewer/log-table.js:103]: Does it play well with both dark and light themes ?

Comment (jonenst) [reply src/components/ReportViewer/log-table.js:103]: the answer is no: 
Capture d’écran de 2022-11-10 10-35-04

Comment (niconoir) [reply src/components/ReportViewer/log-table.js]: Modified to use colors defined for selected rows in themes.

**Review by laurgarn — APPROVED** [review]
Works

**Review by jonenst — APPROVED** [review]

**Pull request merged**